resources/level_1.json

```diff
@@ -2,6 +2,7 @@
     "room": "level 1",
     "size": 18,
     "exit_text": "You have unlocked the exit and proceed through it into a dark and ominous hallway",
+	"description": "The room is dark and damp. You can see a giant padlock on the other door and in the distance a big chest.",
```

Comment:
Not sure what happened here, but I'm gonna be picky and ask you to un-indent these, because Python :)